app/src/main/kotlin/com/flxrs/dankchat/preferences/upload/RecentUploadsAdapter.kt

```diff
@@ -24,6 +30,16 @@ class RecentUploadsAdapter : ListAdapter<RecentUpload, RecentUploadsAdapter.Uplo
             image.load(item.imageUrl) {
                 scale(Scale.FILL)
             }
+            with(copyButton) {
+                setOnClickListener {
+                    val clipData = ClipData.newPlainText("Image URL", item.imageUrl)
+                    context.getSystemService<ClipboardManager>()?.setPrimaryClip(clipData)
+                    // show copied toast only on Android < 13
+                    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.S_V2) {
```

Comment:
Android 13 is `Build.VERSION_CODES.TIRAMISU`, not `S_V2`

Reply:
I went pepega and forget the `=`, it's supposed to be `<=` FeelsDankMan